store/store.go

```diff
@@ -77,6 +77,7 @@ type PostStore interface {
 type UserStore interface {
 	Save(user *model.User) StoreChannel
 	Update(user *model.User, allowRoleUpdate bool) StoreChannel
+	UpdateUpdateAt(userId string) StoreChannel
```

Comment:
Yah, I'm working on refactoring due to a bug that came up I originally didn't see. Should be ready soon.